build.sc

```diff
@@ -18,6 +18,10 @@ object v {
   val bc = ivy"org.bouncycastle:bcprov-jdk15to18:latest.integration"
   val spire = ivy"org.typelevel::spire:latest.integration"
   val evilplot = ivy"io.github.cibotech::evilplot:latest.integration"
+
+  val scalatest = ivy"org.scalatest::scalatest:3.2.0"
+  val scalapar = ivy"org.scala-lang.modules::scala-parallel-collections:1.0.4"
+  val oslib =  ivy"com.lihaoyi::os-lib:0.9.1"
```

Comment:
arithmetic doesn't need this anymore.

Reply:
These 3 dependencies are to enable testfloat test for FP Module for now.

Reply:
test should only exist arithmetic